File: umbrella-insurance-react-ui/src/models/gameTypes/athletic/goal/soccer/v1/Soccer.ts
import { GameType } from "../../../../v1/GameType";

export class Soccer extends GameType {
    static NUMBER_OF_TEAM_MEMBERS?: number = 11;
    static IS_MIXED_SEX_COMPETITION?:boolean = false;
    static IS_CONTACT?:boolean = true;
    static IS_TEAM?:boolean = true;
    static TYPE?: string = "ball";
    static equipment?:string[]  = ["soccer ball", "shin pads"];
    static VENUE?:String = "soccer field";
    static NUMBER_OF_TEAMS?: number = 2;
    static HAS_COIN_TOSS?: boolean = true;
}

File: umbrella-insurance-core/src/main/java/com/umbrella/insurance/core/models/gameTypes/athletic/goal/soccer/v1/Soccer.java
package com.umbrella.insurance.core.models.gameTypes.athletic.goal.soccer.v1;

import com.umbrella.insurance.core.models.entities.GameType;
import lombok.Data;

@Data
public class Soccer extends GameType {
    static int NUMBER_OF_TEAM_MEMBERS = 11;
    static boolean IS_MIXED_SEX_COMPETITION = false;
    static boolean IS_CONTACT = true;
    static boolean IS_TEAM = true;
    static String TYPE = "ball";
    static String[] equipment = {"soccer ball", "shin pads"};
    static String VENUE = "soccer field";
    static int NUMBER_OF_TEAMS = 2;
    static boolean HAS_COIN_TOSS = true;

}
